src/components/listitems/ListItemHeader.tsx

```diff
@@ -0,0 +1,138 @@
+import { Badge } from "../badge";
+
+type ButtonLinkActionProps = {
+  type: "buttonLink";
+  componentProps: ComponentProps<typeof ButtonLink>;
+};
+
+type IconButtonActionProps = {
+  type: "iconButton";
+  componentProps: ComponentProps<typeof IconButton>;
+};
+
+type BadgeProps = {
+  type: "badge";
+  componentProps: ComponentProps<typeof Badge>;
+};
+
+type EndElementProps =
+  | ButtonLinkActionProps
+  | IconButtonActionProps
+  | BadgeProps;
+
+export type ListItemHeader = WithTestID<{
+  label: string;
+  numberOfLines?: number;
+  icon?: IOIcons;
+  endElement?: EndElementProps;
+  // Accessibility
+  accessibilityLabel?: string;
+}>;
+
+export const ListItemHeader = ({
+  label,
+  icon,
+  endElement,
+  accessibilityLabel,
+  testID
+}: ListItemHeader) => {
+  const theme = useIOTheme();
+
+  const listItemAccessibilityLabel = useMemo(
+    () => (accessibilityLabel ? accessibilityLabel : `${label}`),
+    [label, accessibilityLabel]
+  );
+
+  const itemInfoTextComponent = useMemo(
+    () => (
+      <View
+        accessible={endElement === undefined ? true : false}
+        importantForAccessibility={
+          endElement === undefined || endElement.type === "badge"
+            ? "yes"
+            : "no-hide-descendants"
+        }
+        accessibilityElementsHidden={
+          endElement !== undefined && endElement.type !== "badge"
+        }
+      >
+        <H6 weight="Regular" color={theme["textBody-tertiary"]}>
+          {label}
+        </H6>
+      </View>
+    ),
+    [label, theme, endElement]
+  );
+
+  const listItemInfoAction = useCallback(() => {
+    if (endElement) {
+      const { type, componentProps } = endElement;
+
+      switch (type) {
+        case "buttonLink":
+          const buttonLinkAccessibilityLabel = `${listItemAccessibilityLabel}; ${componentProps.accessibilityLabel}`;
+
+          return (
+            <ButtonLink
+              {...componentProps}
+              accessibilityLabel={buttonLinkAccessibilityLabel}
+            />
+          );
+        case "iconButton":
+          const iconButtonAccessibilityLabel = `${listItemAccessibilityLabel}; ${componentProps.accessibilityLabel}`;
+          return (
+            <IconButton
+              {...componentProps}
+              accessibilityLabel={iconButtonAccessibilityLabel}
+            />
+          );
+        case "badge":
+          return <Badge {...componentProps} />;
+        default:
+          return <></>;
+      }
+    }
+    return <></>;
+  }, [endElement, listItemAccessibilityLabel]);
+
+  return (
+    <View
+      style={IOListItemStyles.listItem}
+      testID={testID}
+      accessible={endElement === undefined ? true : false}
+      accessibilityLabel={listItemAccessibilityLabel}
+    >
+      <View style={IOListItemStyles.listItemInner}>
+        {icon && (
+          <View style={{ marginRight: IOListItemVisualParams.actionMargin }}>
+            <Icon
+              name={icon}
+              color="grey-450"
```

Comment:
```suggestion
              color={theme["icon-decorative"]}
```
What about adding a reference to the `icon-decorative` key from the `IOTheme` object? (I know the other list items should be updated, but let's start with this one)